public/generate.go

```diff
@@ -0,0 +1,8 @@
+// Copyright 2016 The Gitea Authors. All rights reserved.
```

Comment:
Please use the current year for new file.
```suggestion
// Copyright 2019 The Gitea Authors. All rights reserved.
```

Reply:
@solderjs I didn't specified but that for all the new files.